webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -0,0 +1,171 @@
+  const dispatch = useDispatch();
+  const [activeTab, setActiveTab] = useState(0);
+  const response = useSelector(state =>
+    selectAPIResponse(state, 'HOST_DETAILS')
+  );
+  useEffect(() => {
+    dispatch(
+      get({
+        key: 'HOST_DETAILS',
+        url: foremanUrl(`/api/hosts/${match.params.id}`),
+      })
+    );
+  }, [match.params.id, dispatch]);
+
+  useEffect(() => {
+    document.body.classList.add('transprenet-border');
+  }, []);
+
+  const handleTabClick = (event, tabIndex) => {
+    setActiveTab(tabIndex);
+  };
+  return (
+    <>
+      <PageSection className="header" variant={PageSectionVariants.light}>
+        <div style={{ marginLeft: '18px', marginRight: '18px' }}>
+          <Breadcrumb style={{ marginTop: '15px' }}>
+            <BreadcrumbItem to="/hosts">Hosts</BreadcrumbItem>
+            <BreadcrumbItem isActive>
+              {response.name || <Skeleton />}
+            </BreadcrumbItem>
+          </Breadcrumb>
+          {/* TODO: Replace all br with css */}
+          <br />
+          <br />
+          <Grid>
+            <GridItem span={2}>
+              <Title headingLevel="h5" size="2xl">
+                {/* TODO: Make a generic Skeleton HOC (withSkeleton) */}
+                {response.name || <Skeleton />}
+              </Title>
+            </GridItem>
+            <GridItem style={{ marginTop: '5px', marginLeft: '10px' }} span={8}>
+              <Badge key={1}>{response.operatingsystem_name}</Badge>{' '}
+              <Badge key={21}>{response.architecture_name}</Badge>
+            </GridItem>
+            <GridItem span={2}>
+              <ActionsBar hostName={response.name || <Skeleton />} />
+            </GridItem>
+          </Grid>
+          <Text style={{ fontStyle: 'italic' }} component={TextVariants.p}>
+            {/* TODO: extracting text and remove timeago usage in favor i18n */}
+            {response.name ? (
+              <div>
+                created{' '}
+                <RelativeDateTime
+                  date={response.created_at}
+                  defaultValue="N/A"
+                />{' '}
+                by {response.owner_name} (updated{' '}
+                <RelativeDateTime
+                  date={response.updated_at}
+                  defaultValue="N/A"
+                />
+                )
+              </div>
+            ) : (
+              <Skeleton width={400} />
+            )}
+          </Text>
+          <br />
+        </div>
+      </PageSection>
+      <Tabs
+        style={{
+          marginLeft: '-18px',
+          marginRight: '-18px',
+          background: 'white',
+        }}
+        activeKey={activeTab}
+        onSelect={handleTabClick}
+      >
+        <Tab eventKey={0} title="Details">
+          <div id="test">
+            <br />
+            <Grid>
+              <GridItem offset={3} span={4}>
+                <StatusAlert
+                  status={response ? response.global_status_label : null}
+                />
```

Comment:
it feels like this status is taking too much space,
I think it should be on the page header line with the host name, and maybe some smaller way to show the status,
this component looks a bit like our toast notifications alerts. 
![badge](https://user-images.githubusercontent.com/26363699/87457372-ed3a0380-c610-11ea-8ddf-fa7955bf2740.png)

Reply:
The status is going to change to a multiple statuses card (we have more statuses to show, and the goal is to create aggregation card for it)